components/Book.vue

```diff
@@ -14,24 +14,32 @@ export default {
   props: ['title', 'author', 'description', 'img', 'id']
 }
 </script>
-
 <style scoped>
-.single-book {
-  display: flex;
-  flex-flow: column;
-  justify-content: center;
-  text-align: center;
-  padding: 10px;
+  .single-book {
+    background-color: #fafafa;
+    display: grid;
+    grid-template-columns: auto;
+    margin: 10px;
+    border-radius: 5px;
+    box-shadow: #AAAAAA 1px 3px;
+  }
+  .book-img {
+    background-position: center;
+    background-size: cover;
+    width: 100%;
+    height: 200px;
```

Comment:
You can't set both width and height value for img, because it would change it's original proportions